pkgdb2/api/packages.py

```diff
@@ -194,17 +199,21 @@ def api_package_edit():
     output = {}
 
     pkg_status = pkgdblib.get_status(SESSION, 'pkg_status')['pkg_status']
```

Comment:
It's not directly related to this change, but it pops out at me seeing ``namespaces`` next to it.  This should probably be ``pkg_statuses`` to indicate that it is plural, a collection, a list.